src/routes/user.js

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+import userValidator from '../middlewares/user-validator';
+
+const user = express.Router();
+
+const middlewares = [userValidator.signUp];
```

Comment:
`userValidator.signUp` is already an array. No need to put it inside another array called `middlewares`. You can easily pass it directly before the controller in place of the `middlewares` array.

Reply:
yeah but there will be more middleware to this route
like that of JWT and any other